src/main/java/com/easypost/model/Rating.java

```diff
@@ -0,0 +1,28 @@
+package com.easypost.model;
+
+import java.util.List;
+import java.util.Map;
+import java.util.HashMap;
+
+import com.easypost.exception.EasyPostException;
+import com.easypost.net.EasyPostResource;
+
+public class Rating extends EasyPostResource {
+	List<?> ratings;
+	List<ShipmentMessage> messages;
+
+	public List<?>  getRating() {return ratings;}
+	public void setRatingList() {this.ratings = ratings;}
```

Comment:
I think this should just be called "setRatings" if the field is called "ratings"